extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -662,6 +662,11 @@ export class AmpA4A extends AMP.BaseElement {
               })
               .then(status => {
                 this.protectedEmitLifecycleEvent_('adResponseValidateEnd');
+                if (getMode().localDev &&
```

Comment:
I think you might want these to be nested instead of joined by `&&`, to ensure that the block is compiled out of production builds. But I could be wrong; I don't know much about how that compiler pass works. Maybe ask Erwin.